package/MDAnalysis/coordinates/LAMMPS.py

```diff
@@ -469,8 +469,9 @@ class DumpReader(base.ReaderBase):
     .. versionadded:: 0.19.0
     """
     format = 'LAMMPSDUMP'
-    _conventions = ["auto", "unscaled", "scaled", "unwrapped",
-                    "scaled_unwrapped"]
+    _conventions = ["auto", "unscaled", "scaled", "unwrapped", 
```

Comment:
Loads of formatting changes and PEP8 bot is complaining a fair bit. I think you will have to revert a few of these to make it happy.